### PR TITLE
Update from package template

### PR DESCRIPTION
**Pull request by Cadair**
moar template (also effectively reverts #127)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,21 +9,22 @@ python-source = "python"
 
 [project]
 name = "streamtracer"
-version = "2.0.1"
 description = "Python wrapped fortran to calculate streamlines"
```
Comment (nabobalis) [inline pyproject.toml]: Is it time to update this?

File: pyproject.toml [hunk 2]
```diff
@@ -9,21 +9,22 @@ python-source = "python"
...
-    {name = "David Stansby", email="dstansby@gmail.com"},
-    {name = "Lars Mejnertsen"}
-]
-readme = "README.md"
+readme = { file = "README.md", content-type = "text/x-markdown" }
 requires-python = ">=3.10"
 license = {text = "GPL v3"}
+authors = [
+  {name = "The SunPy Developers", email="sunpy@googlegroups.com"},
+  {name = "David Stansby", email="dstansby@gmail.com"},
```
Comment (nabobalis) [inline pyproject.toml]: Should we remove his email address? I doubt he wants to be emailed about this anymore

File: pyproject.toml (hunk 1 above)
Comment (Cadair) [reply pyproject.toml]: We will have to update this at release time (I can't find a dynamic solution for maturin).

File: pyproject.toml (hunk 2 above)
Comment (Cadair) [reply pyproject.toml]: @dstansby thoughts?

**Review by nabobalis — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (nabobalis) [reply pyproject.toml]: Oh I meant the description, not the version.

File: pyproject.toml (hunk 2 above)
Comment (dstansby) [reply pyproject.toml]: 👍 to removing my email, would appreciate staying as an author if that's possible though? (but no worries if not)

Comment (nabobalis) [reply pyproject.toml]: Yeah, you should and will stay as an author. I just wanted to avoid any unwanted emails you might get.

Comment (dstansby) [reply pyproject.toml]: Fabulous, thanks :)

File: pyproject.toml (hunk 1 above)
Comment (nabobalis) [inline pyproject.toml]:
```suggestion
description = "Python library to calculate streamlines"
```

File: pyproject.toml (hunk 2 above)
Comment (nabobalis) [inline pyproject.toml]:
```suggestion
  {name = "David Stansby"},
```

**Pull request merged**